bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Hash.cs

```diff
@@ -11,9 +11,92 @@ namespace ElectionGuard
     /// </summary>
     public static class Hash
     {
+        /// <Summary>
+        /// Get the prefix for the hash of an election manifest
+        /// </Summary>
+        public static string Prefix00

```

Comment:
this is a bit convoluted, but it felt like providing this infra across the boundary was better than trying to maintain the values in both c/c++ and c#. maintaining things in two places has burned us in the past